packages/baby-jubjub/tests/index.test.ts

```diff
@@ -31,10 +32,12 @@ describe("BabyJubjub", () => {
 
     it("Should pack a point", async () => {
         const packedPoint = packPoint(publicKey)
+        const strippedPackedPoint = packedPoint & ~(BigInt(1) << BigInt(255))
```

Comment:
I feel like what's being checked here is tricky enough to be worth a comment explaining what it's doing.

Reply:
Right, let me add it!